features/ui/select/select.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable @next/next/no-img-element */
```

Comment:
Actually I found out later that you can still use the Next.js image component without setting the size and so on by importing the image element into the TS file. You could create a path alias to the public icons folder and then use like this

```
import icon from "@icons/chevron-down"

...

<Image src={icon} />
```

That example might be a bit off as I'm typing from my memory